docs/overview/index.md

```diff
@@ -5,18 +5,40 @@ slug: /
+Cradl is a platform that enables developers to build their own document data capture APIs with deep learning.
 
-![How Cradl works illustration](/img/how-cradl-works.png)
+Our approach to data capture is quite different than most, and is centered around a single design goal:
 
-## Structured data extraction
+:::info Design Philosophy
+Be the easiest way for developers to build _customized data capture APIs_ powered by _deep learning models_.
+:::
 
-Every day, everywhere, everyone parses information from documents such as invoices, receipts, price tags and ID cards. The document formats can vary, the information does not. We want to read amounts, dates, names, VAT details, SWIFT codes, invoice and phone numbers. We _know_ the data exists in that image or that PDF, we _just need to parse it_, but doing so requires a human or AI to read it. This is the hallmark of semi-structured data: it lacks a fixed format, but there's valuable information in it _**that can be structured**_.
+## Who is Cradl for?
 
-Cradl's approach to extracting structured data is _**learning by example, not by instruction**_. No need to customize stencils and keywords for every single document type - our machine learning models teach themselves how to read the data you need from your collection of documents. All you need to do is upload enough example data for our models to get going, tell them what to look for and sit back as your custom machine learning model gets trained. Oh - and you don't have to worry about deploying the model, we take care of that, too.
+Cradl is built for developers, and is used by developers e.g. in _financial institutions_, _public sector_, large newspapers and in ERP software companies. If your company:
+
+* process high volumes of semi-structured documents like invoices, receipts, forms, etc.
+* require high-accuracy data capture
+
+then Cradl is for you.
+
+:::tip Need inspiration?
+Check out some of our [use cases](/use-cases)!
+:::
 
-## Getting started
 
-Ready to get started? Try our quickstart guide below, or read more about concepts to get a feel for how the platform works.
+## Key concepts
+Cradl is centered around the following key concepts which you will frequently encounter in these docs:
 
+1. [**Models**](/concepts/models). A model is a _neural network_ that can be trained to solve a specific task, like capturing payment info from invoices.
+2. [**Documents**](/concepts/documents). A semi-structured document in a PDF or image format. Any language and any layout.
+3. [**Datasets**](/concepts/datasets). A dataset consists of many (typically thousands) of documents that are used to train a specific model. 
+4. [**Data bundles**](/concepts/training-data). A data bundle is a collection of one or more datasets and is bound to a specific model.
+5. [**Predictions**](/concepts/predictions). When your model is trained on a dataset, it can start capturing data from new and previously unseen documents.
```

Comment:
```suggestion
5. [**Predictions**](/concepts/predictions). When your model has finished training using your **Data Bundle** it can start capturing data from new and previously unseen documents.
```